README.md

```diff
@@ -1,13 +1,99 @@
-# README
+# GitReps Community
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+## Installation and Usage Instructions
+
+## 1: Clone this repository
+
+```
+git clone https://github.com/GitRepsCommunity/platform.git
+```
+
+## 2: System Dependencies:
+To run this Rails project as intended you will need to install:
+- Ruby
+- Ruby on Rails
+- A Ruby version manager (optional but recommended)
+   - RVM
+   - chruby
+   - rbenv
+   - asdf
+
+## 3: HomeBrew installation:
+
+### Install Ruby:
+```
+brew install ruby
+```
+
+### Verifying Ruby version:
+Most Ruby projects will contain a .ruby-version file with the expected version of Ruby to run.
+You can change your version of Ruby using a version manager like RVM.
+
+### Installing a version manager for Ruby (RVM):
```

Comment:
Since people might want to use different version managers, we might just link to the documentation for installation instructions. Otherwise, we'll have to make sure these steps stay up to date.